### ISSUE TITLE
Unable to find Hash Help. 

### ISSUE THREAD

**Issue by vincentlyy15** [MetaMask/metamask-extension#4658]
Okay so i've recently gotten meta mask and the first couple of times were completely fine but after a day or so, i tried sending ether to my coinbase wallet, but no matter how many times i try to send it wouldn't work 

![eth](https://user-images.githubusercontent.com/40560249/41837070-d198217c-789f-11e8-835a-06df4d074122.PNG)
 
And i would never get the Hash code 
![eth scan](https://user-images.githubusercontent.com/40560249/41837118-f5ccc2aa-789f-11e8-9b81-d4d9333eec9b.PNG)
so basically my ether wont send and i dont know what to do. 
Thanks

**Comment by lazaridiscom** [comment]
Had you increased the gas when retrying?

(Please try additionally support@metamask.io if you don't get a response here)

**Comment by vincentlyy15** [comment]
How much should i increase it by?

**Comment by lazaridiscom** [comment]
I remember a few times I was lazy, and simply doubled the amount when retrying.
